openfisca_france/model/prestations/prestations_familiales/base_ressource.py

```diff
@@ -168,7 +166,9 @@ def nb_enf(famille, period, age_min, age_max):
     Renvoie le nombre d'enfant au sens des allocations familiales dont l'âge est compris entre ag1 et ag2
     """
 
-    period = period.this_month
+    assert period.unit == u'month'
+    assert period.size == 1
```

Comment:
Because nb_enf is a helper function that is used in other modules. Do you see a better solution ?